### ISSUE TITLE
PyPi deploy: Add option for universal wheels

### ISSUE THREAD

**Issue by wolfy1339** [travis-ci/travis-ci#8471]
Add an option in .travis.yml in PyPi deploy to make a universal wheel (Python 2 and 3)

**Comment by ryanhiebert** [comment]
To get `bdist_wheel` to build a universal wheel you need to add a `setup.cfg` that has the contents:

```ini
[bdist_wheel]
universal = 1
```

Then make sure you have `bdist_wheel` in the deploy distributions section of your `.travis.yml`.

Check out my project [Tox-Travis](https://github.com/tox-dev/tox-travis) if you'd like an example. In my case I'm _only_ building a universal wheel.

Is that sufficient for your needs?

**Comment by wolfy1339** [comment]
Yes, that shoud be fine. It would still be a nice to have if they added an option to .travis.yml
